Sources/ARSAL_Time.c

```diff
@@ -122,7 +163,12 @@ int ARSAL_Time_GetLocalTime (struct timespec *res, struct tm *localTime)
         /* No else --> Do nothing if res is NULL */
         if (localTime != NULL)
         {
+#ifdef _WIN32
+            struct tm *tmpLocalTime = localtime(&(ts.tv_sec));
```

Comment:
According to [msdn](https://msdn.microsoft.com/en-us/library/bf12f0hc(VS.80).aspx), the return value of this function is thread_local.